### ISSUE TITLE
SensorBlocks

### ISSUE THREAD

**Issue by Battlepixel** [tsunamayo/Starship-EVO#5448]
If game proof the condition every (set time), please make the sensor blocks activatable to make more efficient building possible. In mechanism that uses multiple Sensorblocks with low loop periodTime (for needed precision) it would be nice to activate the all the condition-loops only when needed with a switch or for example one single sensor with high loopPeriodTime, to check if an entity is nearby, and the functionality of the system is necessary at all.

**Comment by tsunamayo** [comment]
@Battlepixel Hi, sorry I am not sure to understand the request.
Are you asking to be able to be able to deactivate sensor?
I am confused with the low/high period time stuff.
If that is that then yes it should be there, feels more like a bug.
Cheers

**Comment by Battlepixel** [comment]
Yes I wasnt able to connect other logicblocks with a sensor to deactivate it.

**Comment by Uncle-Ulty** [comment]
You don't need to deactivate it. Send the block.'s signal to a "and" logic, and send another signal (switch signal) to this block (on or off). If the switch is on, the sensor gonna send a signal. If the switch is off, the sensor's signal won't do anything.
